projects/core/src/lib/pages/viewer-app/viewer-app.component.ts

```diff
@@ -72,6 +74,14 @@ export class ViewerAppComponent implements OnInit, OnDestroy {
       .subscribe(bookmark => {
           this.router.navigate([], { relativeTo: this.route, fragment: bookmark, replaceUrl: true });
       });
+
+    this.store$.select(selectViewerTitle)
+      .pipe(takeUntil(this.destroyed))
+      .subscribe(title => {
```

Comment:
Maybe we should also reset the title? Something like:

```
.subscribe({
    next: (title) => { this.document.title = title || [DEFAULT_TITLE] },
    complete: () => { this.document.title = [DEFAULT_TITLE] } 
});
```

Reply:
make sense